cmd/erasure-multipart.go

```diff
@@ -437,7 +437,7 @@ func (er erasureObjects) newMultipartUpload(ctx context.Context, bucket string,
 		partsMetadata[index].Metadata = userDefined
 	}
 
-	uploadID := mustGetUUID()
+	uploadID := fmt.Sprintf("%s.%s", globalDeploymentID, mustGetUUID())
 	uploadIDPath := er.getUploadIDDir(bucket, object, uploadID)
```

Comment:
```suggestion
	uploadIDPath := er.getUploadIDDir(bucket, object, uploadID)
```

Do not add "deploymentID" to the `uploadID` folder - they do not need to be the same.